### ISSUE TITLE
Style/IndentationWidth should have a consistent style

### ISSUE THREAD

**Issue by ptarjan** [bbatsov/rubocop#2849]
We'd like to have something like these:

```
Style/IndentHash:
  EnforcedStyle: consistent
Style/IndentArray:
  EnforcedStyle: consistent
Style/FirstParameterIndentation:
  EnforcedStyle: consistent
Style/MultilineOperationIndentation:
  EnforcedStyle: indented
Style/MultilineMethodCallIndentation:
  EnforcedStyle: indented
Style/AlignParameters:
  EnforcedStyle: with_fixed_indentation
```

but would change this code:

```
foo(
  bar, baz do
         1
       end
)
```

into this

```
foo(
  bar, baz do
    1
  end
)
```

**Comment by jonas054** [comment]
`Style/IndentationWidth` checks the indentation of `1`, in this case, compared to the indentation of the `end` keyword of the block. So the key to a solution lies with the `Lint/BlockAlignment` cop. I think/hope #2777 will solve your problem.

**Comment by ptarjan** [comment]
Yay, it looks like using the `AlignWith: do` would match what I want. It still seems weird to have to use another cop instead of `Style/IndentationWidth` having an option for it...

**Comment by jonas054** [comment]
OK, so we don't even have to wait for 2777. Good. Yes, the solution seems a bit backwards in this case, but since you found it I'll close the issue. Making the behavior more intuitive is possible but not a high priority, I would say.

**Comment by ptarjan** [comment]
Why don't we have to wait for #2777?

**Comment by jonas054** [comment]
I misunderstood. We do have to wait.

**Comment by Drenmi** [comment]
@jonas054: I believe this was solved in https://github.com/bbatsov/rubocop/pull/2900.

**Comment by jonas054** [comment]
Yes, with the configuration
```yaml
Lint/BlockAlignment:
  AlignWith: start_of_block
```
running `rubocop --auto-correct` arrives at the indentation @ptarjan requested.
